SETUP.md

```diff
@@ -49,6 +49,28 @@ On Mac OSX and Linux, use `./gradlew` instead of `gradlew`.
 
 If you have errors with a package missing please make sure you have setup your environment, and are using Oracle JDK 8.
 
+To check which java you are using do 
+`java -version` in a command prompt or terminal.
+If you are using OpenJDK 8+ it will not work because the Java distributions above JDK 8 using may not have the needed javax classes.
```

Comment:
Shouldn't this be OpenJDK 9+, to resemble OpenJDK 9 and above not working. I read this as "If you are using OpenJDK 8 and above, which from my own experience, openjdk 8 works fine

Reply:
What I mean is "If you use anything above JDK 8 it MIGHT not work"